client/src/components/Dashboard.test.js

```diff
@@ -3,8 +3,8 @@ import { render, fireEvent, waitForElement } from 'react-testing-library';
 
 import Dashboard from './Dashboard';
 
-describe('<Dashboard />', () => {
-  it('resets balls and strikes to 0 when a player reaches 3 strikes', async() => {
+describe('<Dashboard />', ( ) => {
+  it('reset balls and strikes to 0 when a player reaches 3 strikes', async() => {
     const { getByText } = render(<Dashboard />);
 
     await waitForElement(() => getByText('Strikes: 0'));
```

Comment:
like we discovered in last night's standup, waitForElement is only necessary when performing an asynchronous function that results in the UI change you're testing--since state changes are not async, we can avoid having to use async/await and waitForElement methods in these test